app/routes/user.js

```diff
@@ -3,8 +3,9 @@ const router = express.Router()
 const userController = require('../controllers/user')
 const auth = require('../middleware/auth')
 const isUnderMaintenance = require('../middleware/maintenance')
-// const email = require('../middleware/email')
```

Comment:
why this is removed?

Reply:
It's not removed! Just added a new line by mistake I guess!
![Screenshot from 2020-11-05 22-06-50](https://user-images.githubusercontent.com/17141301/98269062-4f998400-1fb3-11eb-98c1-cf257378f943.png)

Reply:
Oh cool!